### PR TITLE
fix cannot read property id of undefined

### PR DESCRIPTION
**Pull request by TheAndrey777**
add undefined check

### CODE REVIEW

File: lib/plugins/inventory.js [hunk 1]
```diff
@@ -340,11 +340,13 @@ function inject (bot, { version, hideErrors }) {
       return
     }
     assert.ok(click.id <= actionId)
-    while (actionId > click.id) {
+    while (click !== undefined && actionId > click.id) {
       onAccepted()
       click = windowClickQueue.shift()
+       }
+    if (click !== undefined){
+        assert.ok(click)
```
Comment (extremeheat) [inline lib/plugins/inventory.js:348]: What’s the purpose of this assert? It’s hidden behind an != undefined

Comment (TheAndrey777) [reply lib/plugins/inventory.js]: yes i should remove it

**Comment by extremeheat** [comment]
Why the close? No longer an issue?

**Comment by extremeheat** [comment]
Ah, you opened #1901

**Pull request closed without merging**